physics/GWD/unified_ugwp.F90

```diff
@@ -309,7 +311,7 @@ subroutine unified_ugwp_run(me, master, im, levs, ak,bk, ntrac, dtp, fhzero, kdt
       &                                     slmsk(:)
 
     real(kind=kind_phys),    intent(out), dimension(:)          :: dusfcg, dvsfcg
-    real(kind=kind_phys),    intent(out), dimension(:)          :: rdxzb
+    real(kind=kind_phys),    intent(out), dimension(:)          :: zmtb, zlwb, zogw, rdxzb
```

Comment:
Since these are given `intent(out)`, they must be given a value. Right now, they are only conditionally given values. It would be good to check all of these `intent(out)` variables (even those that are not changed in this PR).

Reply:
These definitely need to be changed since they cause the warning during compiles